cwms-data-api/src/main/java/cwms/cda/data/dto/TimeSeries.java

```diff
@@ -56,6 +56,9 @@ public class TimeSeries extends CwmsDTOPaginated {
     @Schema(description = "The units of the time series data",required = true)
     String units;
 
+    @Schema(description = "The version date of the time series trace")
+    Timestamp versionDate;
```

Comment:
The version type should also be included given `versionDate` is null for both non-versioned and aggregate use-cases.

Reply:
done